react/react1/week2/react1-week2/src/Main.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useState } from "react";
+import TodoHeader from "./components/TodoHeader";
+import TodoList from "./components/TodoList";
+import './TodoListStyle.css'
+
+const todoArray = [
+    {
+      id: 1,
+      description: "Get out of bed",
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+    },
+  ];
+
+
+
+
+function Main(){
+    const [todos, setTodos]=useState(todoArray)
+    const buttonClick =()=>{
+        const todoArrayLen =todos.length
+        const addTodoList=[
+            {
+              id: todoArrayLen +1,
```

Comment:
this method isn't ideal, it breaks the logic when you remove an item from the middle of the array, sometimes I use Date.now() when I don't have a database running. 
You can read more about [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now).